src/v2/Apps/Conversation/Components/UnreadMessagesToast.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect } from "react"
+import { ArrowDownIcon, color, Flex, Text } from "@artsy/palette"
+import styled from "styled-components"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { useRouter } from "v2/System/Router/useRouter"
+import usePoll from "../Utils/usePoll"
+
+import { useSystemContext } from "v2/System"
+import { renderWithLoadProgress } from "v2/System/Relay/renderWithLoadProgress"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import {
+  UnreadMessagesToastQuery,
+  UnreadMessagesToastQueryResponse,
+} from "v2/__generated__/UnreadMessagesToastQuery.graphql"
+import { UnreadMessagesToast_conversation } from "v2/__generated__/UnreadMessagesToast_conversation.graphql"
+
+// I assume this will have to be refactored into one of the newer components. Potentially one of the new buttons.
+const Container = styled(Flex)<{ bottomMargin?: boolean }>`
+  background-color: #1023D7; //${color("blue100")};
+  border: none;
+  border-radius: 30px;
+  height: 40px;
+  width: 180px;
+  cursor: pointer;
+  position: fixed;
+  bottom: -40px;
+  transition: all 300ms;
```

Comment:
Random transition speed choice. I'm sure this is standardised.

Reply:
Can we maybe add an `ease-out` ease so it can smooth in?